<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="1"
            tests="2">

  <testsuite name="sample"
    errors="0" failures="1" hostname="" id="" package="sample" skipped="1"
    tests="2" time="0.02000" timestamp="Mon, 24 Jul 2017 05:58:51 GMT">
  
    <testcase name="demoTest" classname="sample" time="0.02000" assertions="2">

      <failure message="Testing if element &lt;#badElement&gt; is present.">    at Object.demoTest (/Users/Sneha/Documents/nightwatch-0.9.16/test/sampletests/withfailures/sample.js:15:15)
    at Module.call (/Users/Sneha/Documents/nightwatch-0.9.16/lib/runner/module.js:62:34)
    at /Users/Sneha/Documents/nightwatch-0.9.16/lib/runner/testcase.js:70:29
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)</failure>

    
    </testcase>
  

  

  
    
    <testcase
      name="demoTest2" classname="sample">
      <skipped />
    </testcase>
    
  
  </testsuite>
</testsuites>
